### PR TITLE
🌐 [i18n-KO] Translated `package_reference/tensorboard.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `package_reference/tensorboard.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/package_reference/tensorboard.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# TensorBoard logger[[tensorboard-logger]]
+
+TensorBoard는 기계학습 실험을 위한 시각화 도구입니다. 주로 손실 및 정확도와 같은 지표를 추적 및 시각화하고, 모델 그래프와 
+히스토그램을 보여주고, 이미지를 표시하는 등 다양한 기능을 제공합니다. 또한 TensorBoard는 Hugging Face Hub와 잘 통합되어 있습니다. 
+TensorBoard 추적(`tfevents` 같은)을 Hub에 푸시하면 Hub는 이를 자동으로 감지하여 시각화 인스턴스를 시작합니다. 
+TensorBoard와 Hub의 통합에 대한 자세한 정보는 [this guide](https://huggingface.co/docs/hub/tensorboard)를 확인하세요.
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/tensorboard.md:10]:
```suggestion
TensorBoard와 Hub의 통합에 대한 자세한 정보는 [가이드](https://huggingface.co/docs/hub/tensorboard)를 확인하세요.
```

Comment (SeungAhSon) [inline docs/source/ko/package_reference/tensorboard.md:10]:
```suggestion
TensorBoard와 Hub의 통합에 대한 자세한 정보는 [가이드](https://huggingface.co/docs/hub/tensorboard)를 확인하세요.
```

File: docs/source/ko/package_reference/tensorboard.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# TensorBoard logger[[tensorboard-logger]]
+
+TensorBoard는 기계학습 실험을 위한 시각화 도구입니다. 주로 손실 및 정확도와 같은 지표를 추적 및 시각화하고, 모델 그래프와 
+히스토그램을 보여주고, 이미지를 표시하는 등 다양한 기능을 제공합니다. 또한 TensorBoard는 Hugging Face Hub와 잘 통합되어 있습니다. 
+TensorBoard 추적(`tfevents` 같은)을 Hub에 푸시하면 Hub는 이를 자동으로 감지하여 시각화 인스턴스를 시작합니다.
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/tensorboard.md:9]:
```suggestion
`tfevents` 같은 TensorBoard 추적을 Hub에 푸시하면 Hub는 이를 자동으로 감지하여 시각화 인스턴스를 시작합니다.
```
괄호가 어색한 것 같아 수정해봤습니다!

**Review by SeungAhSon — COMMENTED** [review]
좋은 번역 감사합니다!

괄호랑 가이드 표시 부분 수정했습니다.

File: docs/source/ko/package_reference/tensorboard.md (hunk 1 above)
Comment (fabxoe) [reply docs/source/ko/package_reference/tensorboard.md:10]: 감사합니다!

File: docs/source/ko/package_reference/tensorboard.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# TensorBoard logger[[tensorboard-logger]]
```
Comment (harheem) [inline docs/source/ko/package_reference/tensorboard.md:5]:
```suggestion
# TensorBoard 로거[[tensorboard-logger]]
```

Comment (harheem) [reply docs/source/ko/package_reference/tensorboard.md:5]: 타이틀을 로거로 해두셔서 통일성을 위해 제안해봅니다!

File: docs/source/ko/package_reference/tensorboard.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+TensorBoard는 기계학습 실험을 위한 시각화 도구입니다. 주로 손실 및 정확도와 같은 지표를 추적 및 시각화하고, 모델 그래프와 
+히스토그램을 보여주고, 이미지를 표시하는 등 다양한 기능을 제공합니다. 또한 TensorBoard는 Hugging Face Hub와 잘 통합되어 있습니다. 
+TensorBoard 추적(`tfevents` 같은)을 Hub에 푸시하면 Hub는 이를 자동으로 감지하여 시각화 인스턴스를 시작합니다. 
+TensorBoard와 Hub의 통합에 대한 자세한 정보는 [this guide](https://huggingface.co/docs/hub/tensorboard)를 확인하세요.
+
+이 통합을 위해, `huggingface_hub`는 로그를 Hub로 푸시하기 위한 사용자정의 로거를 제공합니다.
```
Comment (harheem) [inline docs/source/ko/package_reference/tensorboard.md:12]:
```suggestion
이 통합을 위해, `huggingface_hub`는 로그를 Hub로 푸시하기 위한 사용자 정의 로거를 제공합니다. 
```

**Comment by jungnerd** [comment]
@Wauplin LGTM! I think this PR is good to go!

**Review by Wauplin — APPROVED** [review]
Thank you!

**Pull request merged**